docs/OnTrack/Jupyter Notebook/Docker-Documentation-Research-T1-2022.md

```diff
@@ -1,32 +1,67 @@
 # Docker Documentation and Research (WIP) T1-2022
 
-- [Back to Jupyter Notebook Documentation Index](Index.md)
+[Back to Jupyter Notebook Documentation Index](Index.md)
 
-## Conversion Relating to Docker Processes
+## Intro
+
+As we know, the Jupyter Notebook conversion feature will happen via processes that occur inside
+Docker containers. In this document we will discuss the main architecture of this structure in
+relation to OnTrack.
+
+OnTrack is deployed with two main containers that will always be running:
+
+- the frontend container which hosts the logic behind the user interface of OnTrack (mostly
+  unrelated to this feature)
+- the backend container which hosts the logic related to database interactions and other processes
```

Comment:
To me, it might be better to have frontend and backend capitalized and maybe have them as 'Front-end' and 'Back-end' with a '-' in the middle, just to ensure readability. Just a suggestion though.

Reply:
Thanks @mfclark1 , I've made this change, although have kept 'back-end' anmd 'front-end' lowercase.